WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+class ChangeEventOrderAndNames < ActiveRecord::Migration
+  def up
+    # No change for Rubik's Cube event.
```

Comment:
Just to make it clear, how about we do one line at  the end where we set `cellName = name`?